app/src/analysis/initializeCoreDb.ts

```diff
@@ -72,24 +76,29 @@ export async function initializeCoreDb(): Promise<sqlite3.Database> {
   }
   const possibleAddressBookDB = await findPossibleAddressBookDB();
   const lorDB = initializeDB(chatPaths.app);
+
+  // Drop everything LOR specific if it exists
   await dropAllTables(lorDB);
 
-  log.info('Dropped all tables.');
+  // Add the contact name column regardless
+  // It will just be empty if we don't find an address book db table
+  // As a result, we can use COALESCE(contact_name, id)
+  await addContactNameColumn(lorDB);
```

Comment:
Now, by always adding the contact name we should always be able to COALESCE, cc @Teddarific